hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/resources/capacity-scheduler.xml

```diff
@@ -97,6 +97,15 @@
     </description>
   </property>
 
+  <property>
+    <name>yarn.scheduler.capacity.root.decided.reservable</name>
```

Comment:
Why do we touch this file?

Reply:
ListReservations Methods requires the queue to open the Reservation System, so during the junit test, you need to add configuration information.

If the queue does not add this configuration, the following information will be prompted.
```
Caused by: org.apache.hadoop.yarn.exceptions.YarnException: The specified queue: root.decided is not managed by reservation system. Please try again with a valid reservable queue.
```

for configuration information, please refer to the following page
[Configuring_ReservationSystem](https://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html#Configuring_ReservationSystem_with_CapacityScheduler)

Reply:
Considering that I need to complete the following methods in the future, I modified capacity-scheduler.xml in the resource of test·
The methods involved are as follows:
```
getNewReservation
submitReservation
listReservations
updateReservation
deleteReservation
```